### DISCUSSION TITLE
SAML with Keycloak

### DISCUSSION THREAD

**Discussion by EmilaineBorato** [getredash/redash#6400] [General]
Hii,

I'm trying to configure SAML with keycloak but I've been getting some error messages.

![image](https://github.com/getredash/redash/assets/28653612/f9d46cb3-6ab8-47d2-8248-1890ef6736ef)

I'm using the master branch.
Can someone help me?

**Comment by nguyenvinhlinh** [comment]
Hi, this is my blog about configure Redash with Keycloak, may be it could help, it's really long, so I can't re-type it in this discussion while driving
https://hexalink.xyz/redash/2023/07/27/How-to-configure-KeyCloak-to-support-Redash-SAML-SSO.html
